examples/covariance/plot_mahalanobis_distances.py

```diff
@@ -72,73 +83,107 @@
 outliers_cov[np.arange(1, n_features), np.arange(1, n_features)] = 7.
 X[-n_outliers:] = np.dot(np.random.randn(n_outliers, n_features), outliers_cov)
 
-# fit a Minimum Covariance Determinant (MCD) robust estimator to data
+# %%
+# Comparison of results
+# ---------------------
+#
+# Below we fit MCD and MLE based covariance estimators to our data and print
+# the estimated covariance matrices. Note that the estimated variance of
+# feature 2 is much higher with the MLE based estimator (7.5) than
+# that of the MCD robust estimator (1.2). This shows that the MCD based
+# robust estimator is much more resistant to the outlier samples, which were
+# designed to have a much larger variance in feature 2.
+
```

Comment:
Move the `from sklearn.covariance import EmpiricalCovariance, MinCovDet` and matplotlib import here?